scripts/spyder_win_post_install.py

```diff
@@ -21,17 +21,95 @@
 KEY_C1 = KEY_C0 + r"\command"
 
 
+# ability to run spyder-win-post-install outside of bdist_wininst installer
+# copied from pywin32-win-post-install
+try:
+    # When this script is run from inside the bdist_wininst installer,
+    # file_created() and directory_created() are additional builtin
+    # functions which write lines to Python23\pywin32-install.log. This is
+    # a list of actions for the uninstaller, the format is inspired by what
+    # the Wise installer also creates.
+    file_created
+    is_bdist_wininst = True
+except NameError:
+    is_bdist_wininst = False # we know what it is not - but not what it is :)
+    def file_created(file):
+        pass
+    def directory_created(directory):
+        pass

```

Comment:
There is already a bit of explanation from Mark Hammond on [L27](https://github.com/spyder-ide/spyder/pull/2377#discussion-diff-29477366R27). I added a link to bdist_wininst postinstaller in the standard referencce library and additional comments requested by @Nodd  directly prior to the dummy method definitions in 330b0aa.